index.js

```diff
@@ -1 +1,107 @@
-console.log('hello')
\ No newline at end of file
+let gameFlow = document.querySelector('.game')
+let area = document.getElementById("touch-area")
+let scoreElement = document.getElementById('score')
+let scoreCounter = 0 // for score
+
+const gameFlowHeight = parseInt(window.getComputedStyle(gameFlow).getPropertyValue('height'))
+
+const blockClasses = ['left-arrow', 'up-arrow', 'down-arrow', 'right-arrow']
+
+class Block {
+    id = Date.now
+    className = getRandom().itemClass
+
+    createItem() {
+        const newItem = document.createElement('div')
+        newItem.setAttribute('class', `${this.className} drop-block`)
+        newItem.setAttribute('id', `${this.id}`)
+        return newItem
+    }
+}
+
+
+const checkTouch = (item) => {
+    const itemPosition = parseInt(window.getComputedStyle(item).getPropertyValue('top'))
+    const excellentArea = itemPosition >= (gameFlowHeight - 100) + 10 && itemPosition <= gameFlowHeight - 60
+    const goodArea = itemPosition >= (gameFlowHeight - 100) - 10 && itemPosition <= gameFlowHeight - 40
+
+    if (excellentArea) {
+        scoreCounter += 2
+        scoreElement.textContent = scoreCounter
```

Comment:
```scoreElement.textContent = scoreCounter``` is repeated in all the condition blocks, it could be moved out of them. ```excellentArea``` and ```goodArea``` are excluding each other, so ```return``` can be removed from both condition blocks